test/playwright.fixtures.ts

```diff
@@ -24,7 +24,7 @@ import { Connection } from '../lib/rpc/client/connection';
 import { Transport } from '../lib/rpc/transport';
 import { setUnderTest } from '../lib/helper';
 import { installCoverageHooks } from './coverage';
-import { parameters, registerFixture, registerWorkerFixture, registerParameter } from './runner';
+import { parameters, registerFixture, registerWorkerFixture } from './runner';
```

Comment:
I think `parameters` and `matrix` should be named similarly, otherwise it's not clear parameters come from matrix.

Reply:
They aren't in any of the CIs, so let's follow up.